wsdl/ver10/device/wsdl/devicemgmt.wsdl

```diff
@@ -3553,6 +3574,24 @@ IN NO EVENT WILL THE CORPORATION OR ITS MEMBERS OR THEIR AFFILIATES BE LIABLE FO
 			<wsdl:input message="tds:StartFirmwareUpgradeRequest"/>
 			<wsdl:output message="tds:StartFirmwareUpgradeResponse"/>
 		</wsdl:operation>
+		<wsdl:operation name="UpgradeFirmware">
```

Comment:
@ocampana-videotec ,
Method name is "UpgradeFirmware" or "StartCloudFirmwareUpgrade" ?
In the description, image and capability there is a mismatch. 
Please make the description and wsdl definition consistent.